llpc/test/shaderdb/core/OpSDot_TestIVec_lit.spvasm

```diff
@@ -0,0 +1,54 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: %{{[0-9]+}} = call i32 (...) @lgc.create.integer.dot.product.i32(<4 x i32> %{{[0-9]+}}, <4 x i32> %{{[0-9]+}}, i32 0, i32 3)
```

Comment:
Why don't we check for the add intrinsic here?

Reply:
Sounds good. Accept.